lib/fastlane/plugin/firebase_app_distribution/actions/firebase_app_distribution_login.rb

```diff
@@ -55,6 +90,18 @@ def self.authors
       def self.is_supported?(platform)
         [:ios, :android].include?(platform)
       end
+
+      def self.available_options
+        [
+          FastlaneCore::ConfigItem.new(key: :port,
+                                       env_name: "FIREBASEAPPDISTRO_LOGIN_PORT",
+                                       description: "Port for the local web server which receives the response from Google's authorization server",
+                                       default_value: "8080",
```

Comment:
Do you think it's worth adding it logic to find a random open port?

Reply:
I think it would be fine to catch the error if the port is in use and recommend using the --port flag.